src/libraries/System.Text.Json/pkg/System.Text.Json.pkgproj

```diff
@@ -5,6 +5,7 @@
     <ProjectReference Include="..\src\System.Text.Json.csproj">
       <SupportedFramework>net461;netcoreapp2.0;uap10.0.16299;$(AllXamarinFrameworks)</SupportedFramework>
     </ProjectReference>
+    <ProjectReference Include="..\codegenerator\codegenerator.csproj" SkipPackageFileCheck="true" HarvestDependencies="false" />
```

Comment:
You'll be able to remove both once this change is in: https://github.com/dotnet/arcade/pull/5741
```suggestion
    <ProjectReference Include="..\codegenerator\codegenerator.csproj" />
```

Reply:
That change is merged now.

Reply:
We need to ingest the arcade update.

Reply:
https://github.com/dotnet/runtime/pull/39101 - need to merge this, then pull dotnet/runtime over